glustercli/cmd/utils.go

```diff
@@ -55,7 +55,8 @@ func sizeToMb(value string) (uint64, error) {
 	default:
 		size = sizeValue
 	}
-	return size, nil
+	sizeInBytes := size * 1024 * 1024
```

Comment:
instead of converting to MB and then converting back again. Please change in switch itself

```
switch sizeUnit {
 	switch sizeUnit {
 	case "K":
 		size = sizeValue * KB
 	case "G":
 		  size = sizeValue * GB
 	case "T":
 		size = sizeValue * TB
	default:
		size = sizeValue
	}
```

Reply:
Not related to this PR, but can add support for `KiB`, `MiB`, `GiB` and `TiB`

```
switch sizeUnit {
 	switch sizeUnit {
 	case "K", "KiB":
 		size = sizeValue * KiB
        case "KB":
 		size = sizeValue * KB
 	case "G", "GiB":
 		  size = sizeValue * GiB
        case "GB":
 		size = sizeValue * GB
 	case "T", "TiB":
   		size = sizeValue * TB
        case "TB":
 		size = sizeValue * TB
	default:
		size = sizeValue
	}
```